plantio/urls.py

```diff
@@ -2,10 +2,12 @@
 from django.urls import include
 
 from plantio.views.plantio import PlantioAPIView
+from plantio.views.aplicacao import AplicacaoAgrotoxicoAPIView
 
 
 urlpatterns = [
     url(r'^plantio/', include([
         url(r'^$', PlantioAPIView.as_view(), name='plantio-create'),
+        url(r'^aplicar/$', AplicacaoAgrotoxicoAPIView.as_view(), name='plantio-associar'),
```

Comment:
Somente aplicar deixa muito em aberto sobre o que a rota faz. Sugiro especificar, visto que essa rota só faz aplicação de agrotoxico
```suggestion
        url(r'^aplicacao/agrotoxico/$', AplicacaoAgrotoxicoAPIView.as_view(), name='plantio-associar'),
```
```suggestion
        url(r'^aplicar/agrotoxico/$', AplicacaoAgrotoxicoAPIView.as_view(), name='plantio-associar'),
```